### DISCUSSION TITLE
Are there plans to add etna to conda-forge repo?

### DISCUSSION THREAD

**Discussion by ArtemLiA** [etna-team/etna#414] [Q&A] [answered]
Hi everyone!

Are there plans to make it possible to install etna via conda?

**Comment by d-a-bunin** [comment]
Hi!
We haven't thought about this yet. Could you please share why you would like to install etna via conda?
Thanks!

**Reply by ArtemLiA** [reply]
I could install all packages I need when creating environment and manage their versions using Anaconda Prompt.

**Reply by d-a-bunin** [reply, marked as answer]
As far as I remember, you could [install any pip package in conda env](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-pkgs.html).

For now, we have no plans about publishing to conda repos. If you think that this is an important thing to have, you could create an improvement discussion about it.

**Reply by ArtemLiA** [reply]
Yes, you are right. However, packages installed via pip cannot be updated via conda. Anyway, thanks a lot fot the reply! I dont think that this is a very important thing, so most likely I won't create improvement discussion about it.
